### PR TITLE
fix(ngc): don't codegen foo.d.ngfactory.ts from foo.d.ts

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

**Comment by alexeagle** [comment]
note, I considered copying `@angular2-material/button/button.{d.ts,metadata.json,js}` here as a better verification of what we really care about.
But then I thought it would get annoying to keep it up to date.
@jelbourn what do you think about adding `npm install @angular2-material/{core,button}` in the integration test? It will fail in the case where angular has a breaking change and doesn't work with the current material release.

Otherwise I suppose this copy of the BasicComp as a third-party dep is the right shape.

**Comment by jelbourn** [comment]
@alexeagle MdButton doesn't really do a ton that is likely to break in future versions of Angular, so I think it's pretty safe.

**Comment by alexeagle** [comment]
Well, `@angular2-material` alpha7-2 is already broken wrt. angular/angular HEAD:
`Error: Error at /private/var/folders/w6/ts7cfpns233_7_4mkwwm3fcc002p45/T/e2e_test.1471372390/node_modules/@angular2-material/button/button.d.ts:34:44: Generic type 'Type<T>' requires 1 type argument(s).`

File: modules/@angular/compiler-cli/integrationtest/src/basic.thirdparty.d.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export declare class BasicComp {
```
Comment (IgorMinar) [inline modules/@angular/compiler-cli/integrationtest/src/basic.thirdparty.d.ts]: don't we need a license header in this file?

Comment (alexeagle) [inline modules/@angular/compiler-cli/integrationtest/src/basic.thirdparty.d.ts]: removing this file - Jeremy and I decided to make this a material integration test too, but I need (yet) another material release to pick up a bug fix for loading material in node.

**Pull request merged**